### ISSUE TITLE
Installation stalls at database creation

### ISSUE THREAD

**Issue by VBScab** [kimai/kimai#700]
#### Expected behaviour
Installer completes

#### Actual behaviour
Hangs after asking for database user details. The connection attempt never times-out

#### Steps to reproduce the behaviour
Run the installer

#### Kimai version
0.9.3.1384

#### Notes
- Windows firewall is off
- MySQL 5.7
- IIS 7
- PHP v5.4, 5.6 and 7.0 all tried, same result
- phpinfo(0) shows that MySQLi extension is installed and enabled and listening on port 3306
- changed 'mysql_connect' call in 'Installer\processor.php' to 'mysqli_connect'
- Edited to Windows 'hosts' file to include/exclude mapping of "localhost" to "127.0.0.1"

As a quick test, I created a script to test the connection to MySQL:

```
<?php
ini_set('display_errors', 1);
error_reporting(E_ALL);
$con = mysqli_connect("127.0.0.1:3306", "root","");
if (!$con) {
    die('mysqli connection failed: ' . mysqli_connect_error() );
}
?>

```
This also never times-out nor produces any error message. I've tried all manner of connection strings:
```
$con = mysqli_connect("localhost", "root","");
$con = mysqli_connect("localhost:3306", "root","");
$con = mysqli_connect("127.0.0.1", "root","");
$con = mysqli_connect("127.0.0.1:3306", "root","");
$con = mysqli_connect("127.0.0.1", "root","","","3306");

```
Nothing works.

I've created a dedicated user account in MySQL with and without a password - no joy. I created an account with the same credentials as the local Windows account I'm using - no joy. 

I'm at my wit's end, as this should just work :-(

**Comment by kevinpapst** [comment]
Please try the latest development version: 
https://github.com/kimai/kimai/archive/master.zip
and see if that helps.
We made a lot of changes in several places, to support the mysqli extension.

**Comment by VBScab** [comment]
After unZIPping the archive to the IIS VD and restarting everything, it first choked at line 185, saying that 'db_type' was undefined. I stopped and restarted the web site and IIS and it progressed. Finally!

Can I suggest a re-wording of the page where the installing user is asked to choose a database? It needs to be made clearer that one needs to choose one or the other and not both. I appreciate that that "error" is trapped - I tested it! - but a re-wording would make things clearer.

Next, the page presented when the installation has finished, where the user is asked to log in, seems to have not picked up its style(s). What I saw was a very plainly formatted page and the button for submitting the form had no text in it and was tiny. It was only because I saw a faint grey line that I worked out that it was meant to be a button at all.

Lastly, thanks very much for the prompt responses to my posts! :-)
